src/test/java/org/jabref/logic/bibtex/BibEntryWriterTest.java

```diff
@@ -272,8 +273,8 @@ void roundTripWithCamelCasingInTheOriginalEntryAndResultInLowerCase() throws IOE
         String bibtexEntry = OS.NEWLINE + "@Article{test," + OS.NEWLINE +
                 "  Author                   = {Foo Bar}," + OS.NEWLINE +
                 "  Journal                  = {International Journal of Something}," + OS.NEWLINE +
-                "  Note                     = {some note}," + OS.NEWLINE +
```

Comment:
This change is OK for me. `Number` really belongs to a journal. More than `Note`